contentcuration/contentcuration/utils/garbage_collect.py

```diff
@@ -145,3 +148,20 @@ def clean_up_tasks():
         count, _ = TaskResult.objects.filter(date_done__lt=date_cutoff, status__in=states.READY_STATES).delete()
 
     logging.info("Deleted {} completed task(s) from the task table".format(count))
+
+
+@delay_user_storage_calculation
+def clean_up_stale_files(last_modified=now() - datetime.timedelta(days=30)):
```

Comment:
Might be good to define the default inside the function scope. At the moment, by defining it in the function definition, `now()` will be whatever `now()` was at the time the module was imported. So if this function was run from inside a process that had been running for a hundred days, it would actually be looking for `last_modified` that was 130 days ago.

The usual Python pattern for this is to set a default of `None`, and then do `if last_modified is None` inside the function and then set it to the default value.

The big gotcha that can sometimes happen here is if the default value is set to a mutable type like a dict, and then successive invocations of the function can actually modify and share the default value.

See here: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for more about that.

Reply:
That is a good point that I did not realize, thank you! I will make those changes and update the pr.